app/views/home/show.html.erb

```diff
@@ -0,0 +1,7 @@
+<h1><%= t(".welcome") %></h1>
+
+<h2><%= t(".new_account") %></h2>
+
+<form action="/users/new" method="get">
+  <button type="submit"><%= t(".button") %></button>
+</form>
```

Comment:
This form doesn't seem to be posting any params. And it used `get` to post no params.
And it is not covered by tests.

If you are to put a sign up form on the home page, a suggestion to use a partial that is shared between the new user view and this view. Also use `simple_form`

Reply:
Thanks Elle, It was covered by a test that tied into new user at one point, I should have made a note when I changed that test! I'm thinking of a nav bar to render in the layouts/application view that would replace the form, but I will remember to user simple form in the nav bar!